www/src/components/shared/pullquote.js

```diff
@@ -168,13 +168,19 @@ const Pullquote = ({ citation, narrow = false, children }) => {
     <PullquoteRoot narrow={narrow} className={className}>
       {children}
       {citation && <Citation>&mdash; {citation}</Citation>}
-      <QuotationMark
-        dangerouslySetInnerHTML={{ __html: QuotationMarkOrnament }}
-      />
+      <QuotationMark>
+        <QuotationMarkOrnament />
+      </QuotationMark>
       <div>
-        <Star dangerouslySetInnerHTML={{ __html: StarOrnament }} />
-        <Star dangerouslySetInnerHTML={{ __html: StarOrnament }} />
-        <Star dangerouslySetInnerHTML={{ __html: StarOrnament }} />
+        <Star>
```

Comment:
@abhishekjakhar If you're still looking for things to refactor, I think this is a good candidate. We can simplify this and other components by converting these styled components into normal components with the icons embedded in them.

Reply:
@tesseralis this sounds good to me, I will start working on it. Thank you.